### ISSUE TITLE
File upload working on PHP but not on nekotools -server

### ISSUE THREAD

**Issue by ciscoheat** [HaxeFoundation/haxe#3145]
This simple example works on PHP but not when using `nekotools -server`, neither on Windows or Linux:

``` haxe
package ;

#if neko
import neko.Lib;
import neko.Web;
#elseif php
import php.Lib;
import php.Web;
#end
import sys.io.File;

class Main
{
    static function main()
    {
        if (Web.getMethod() == "GET")
        {
            printForm();
        }
        else
        {
            var file = Web.getMultipart(10 * 1024 * 1024);
            var out = File.write('test.jpg');
            out.writeString(file.toString());
            out.close();
        }
    }

    static private function printForm()
    {
        Lib.print('
<form method="POST" action="/" enctype="multipart/form-data">
File: <input type="file" name="content" size="38" /> <input type="submit" />
</form>
        ');
    }
}
```

When using Neko, the file is outputted in the console so the upload itself works, but the output file only contains `{}`. On PHP it will contain the whole uploaded file.

**Comment by ncannasse** [comment]
I haven't much plans to work on nekotools server, although we are accepting PR if someone want to contribute it.

**Comment by sonygod** [comment]
it's seem fixed 3.4
